Features/RoleUI/RoleUI.feature

```diff
@@ -0,0 +1,166 @@
+    And I should see a label for the Role Assigments
+    And I should see a group with the Role Assigments
+      | User/group | Limitation |
+
+  @javascript @edge
+  Scenario: Display the information of a role with empty policies
+    Given I have an "Organizers" role
+    And "Organizers" do not have any assigned policies
+    And "Organizers" do not have any assigned Users and groups
+    When I am viewing the "Organizers" role's details
+    Then I should see that there are no policies set up for this role
+    And I should see that this role has no Role Assigments
+
+  ##
+  #
+  # Creating Roles
+  #
+  ##
+  @javascript @common
+  Scenario: Validate the existence of expected fields when creating a role
+    Given I am on the Roles page
+    When I create a new role
+    Then I should see a "Name" field
+    And I should see a "Save" button
+    And I should see a "Cancel" button
+
+  @javascript @common
+  Scenario: create a valid role
+    Given I am on the Roles page
+    And I create a new role
+    And I set "Organizers" as the "Name"
+    When I click on "Save" button
+    Then the Role is successfully published
+
+  @javascript @edge
+  Scenario: create a valid role with japanese characters
+    Given I am on the Roles page
+    And I create a new role
+    And I set "私は日本の名前で試してみる必要が" as the "Name"
+    When I click on "Save" button
+    Then the Role is successfully published
+
+  @javascript @edge
+  Scenario: creating a Role with an empty name triggers an obligation to fill the name
+    Given I am on the Roles page
+    And I create a new role
+    And I set empty as the "Name"
+    When I click on "Save" button
+    Then I see a message asking for the field "Name" to be filled
+    And the Role is not published
+
+  @javascript @edge
+  Scenario: creating a Role with an existing identifier fails validation
+    Given I have an "Organizers" role
+    And I create a new role
+    And I set "Organizers" as the "Name"
+    When I click on "Save" button
+    Then I see a message saying that the name already exists
+    And the Role is not published
+
+  @javascript @common
+  Scenario: cancel the creation of a role
+    Given I am on the Roles page
+    And I create a new role
+    And I set "Organizers" as the "Name"
+    When I click on "Cancel" button
+    Then the Role is not published
+
+  ##
+  #
+  # Updating Roles
+  #
+  ##
+  @javascript @common
+  Scenario: Validate the existence of expected fields when editing a role
+    Given I have an "Organizers" role
+    When I edit the "Organizers" role
+    Then I should see an "Edit <Organizers>" label
+    And I should see a "Name" field
+    And I should see a "Save" button
+    And I should see a "Cancel" button
+
+  @javascript @common
+  Scenario: Update a role
+    Given I have an "Organizers" role
+    And I edit the "Organizers" role
+    And I set "old_Organizers" as the "Name"
+    When I click on "Save" button
+    Then the Role is successfully published
```

Comment:
Yes it can. In BDD discussions apart from the current one, it was decided to leave the "publish" action in separate scenarios from the "view" action.